sos/collector/clusters/ocp.py

```diff
@@ -69,7 +69,8 @@ class ocp(Cluster):
         ('role', 'master', 'Colon delimited list of roles to filter on'),
         ('kubeconfig', '', 'Path to the kubeconfig file'),
         ('token', '', 'Service account token to use for oc authorization'),
-        ('with-api', False, 'Collect OCP API data from a master node')
+        ('with-api', False, 'Collect OCP API data from a master node'),
+        ('api-url', '', 'Alternate API URL of an external control-plane'),
```

Comment:
I think there is an extra comma in api-url definition.

Reply:
If you mean at the end of line 73, I've been asked by sos maintainers in the past to add it to avoid changing one extra line on subsequent changes